src/pageObject.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the Conduit logo';
+  };
```

Comment:
```suggestion
  clickOnLogo() {
    return 'Click on the Conduit logo';
  };
```
task description - PageObject should contain clickOnLogo method. The method should return a string 'Click on the' + logo from the Header class

Reply:
you don't use header class here